README.md

```diff
@@ -7,5 +7,18 @@ Then run:
 ```shell
```

Comment:
Lets add also some info regarding the goal and the structure of this project.
Like
```
The goal of this project is to give you some basic insights regarding the platform and [neuro-flow](https://neu-ro.gitbook.io/neuro-flow/) engine workings.

Here we show you with help of Pythorch's [MNIST example](https://github.com/pytorch/examples/tree/master/mnist) how to persist datasets, launch trainings and store their results on the Neu.ro platform.

```

Maybe something else.

Reply:
Good idea!